packages/paste-website/src/component-examples/filter-group/constants.ts

```diff
@@ -0,0 +1,70 @@
+import type {TableDataRow} from './types';
+
+export const roomTypes = ['All', 'Group', 'WebRTC Go', 'Peer to Peer'];
```

Comment:
nit: constants tent to be all caps https://github.com/twilio-labs/paste/blob/main/packages/paste-website/src/constants.ts